chain-abci/src/app/query.rs

```diff
@@ -7,6 +7,7 @@ use chain_core::state::account::StakedStateAddress;
 use chain_core::tx::data::{txid_hash, TXID_HASH_ID};
 use integer_encoding::VarInt;
 use parity_codec::{Decode, Encode};
+use std::convert::TryFrom;
```

Comment:
I don't know if this is an unused import. Clippy should warn if it is.

Reply:
it's used -- before it was using that custom function in RedeemAddress, now it's using a trait